i18n/en.json

```diff
@@ -5085,6 +5085,7 @@
   "user.settings.import_theme.submit": "Submit",
   "user.settings.import_theme.submitError": "Invalid format, please try copying and pasting in again.",
   "user.settings.languages.change": "Change interface language",
+  "user.settings.languages.dropdown": "drop down selector to change the interface language",
```

Comment:
we also suffix the translation with "ariaLabel" to help translators know its an aria label.
eg
```
file_upload.menuAriaLabel
insights.filter.ariaLabel

Reply:
Also, this is a bit nitpicking, but "dropdown" should probably be a single word in this case